fem/qinterp/det.cpp

```diff
@@ -75,8 +75,8 @@ static void Det2D(const int NE,
 
    mfem::forall_2D_batch(NE, Q1D, Q1D, NBZ, [=] MFEM_HOST_DEVICE (int e)
    {
-      constexpr int MQ1 = T_Q1D ? T_Q1D : MAX_Q1D;
-      constexpr int MD1 = T_D1D ? T_D1D : MAX_D1D;
+      constexpr int MQ1 = T_Q1D ? T_Q1D : DofQuadLimits::MAX_Q1D;
+      constexpr int MD1 = T_D1D ? T_D1D : DofQuadLimits::MAX_D1D;
```

Comment:
The template parameters `MAX_D1D` and `MAX_Q1D` are not used anymore, so they can be removed.

Reply:
Fixed in [c80186c](https://github.com/mfem/mfem/pull/3722/commits/c80186cecaa1cd3fe4a288d239cf119be85703b5)